src/display/inky_display.py

```
@@ -58,5 +58,12 @@ def display_image(self, image, image_settings=[]):
             raise ValueError(f"No image provided.")
 
         # Display the image on the Inky display
-        self.inky_display.set_image(image)
-        self.inky_display.show()
\ No newline at end of file
+        image_settings_cfg = self.device_config.get_config('image_settings') or {}
+        inky_saturation = image_settings_cfg.get("inky_saturation", 0.5)
+        logger.info(f"Inky Saturation: {inky_saturation}")
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)